server/src/database/schema.prisma

```diff
@@ -554,6 +557,21 @@ model Attachment {
   @@map("attachments")
 }
 
+model Favorite {
+  id          String   @id @default(uuid())
+  workspaceId String?
+  /// @TypeGraphQL.omit(input: true, output: false)
+  person      Person?  @relation(fields: [personId], references: [id])
+  personId    String?
+  /// @TypeGraphQL.omit(input: true, output: false)
+  company     Company? @relation(fields: [companyId], references: [id])
+  companyId   String?
+  User        User?    @relation(fields: [userId], references: [id])
```

Comment:
let's link favorites to a workspaceMember actually

Reply:
Do you want me to replace the `User` with `workspaceMember`?

Reply:
yes! A User (email + password) can be tied to different workspaces through workspaceMember.
Favorites should be attached to worksapceMember

Reply:
It's automatically adding the User relation on save, I don't have `format on save` turned on in my vscode too, any suggestions? Thanks

Reply:
It's your VSCode Prisma extension, you should also update your User model upper in the file and move Favorite to workspaceMember

Reply:
Done! Let me know If I missed something